sample-test-macros/Cargo.toml

```diff
@@ -11,12 +11,12 @@ path = "src/lib.rs"
 proc-macro = true
 
 [dependencies]
-lazy_static = "1.4"
 proc-macro2 = "1.0"
 quote = "1.0"
 sample-std = { path = "../sample-std" }
 syn = { version = "1.0", features = ["full", "extra-traits"] }
 
 [dev-dependencies]
-quickcheck = "1.0"
+once_cell = "1.0"
+lazy_static = "1.4"
```

Comment:
do we need this?

Reply:
They're in the tests, just to verify that the macro can work with them.